Rakefile

```diff
@@ -6,7 +6,7 @@
 # to log level :warn reduces logging and increases execution speed.
 ENV["LOG_LEVEL"] = "warn"
 
-ENV['GOVUK_APP_DOMAIN'] = 'dev.gov.uk' unless ENV['GOVUK_APP_DOMAIN']
+ENV['GOVUK_APP_DOMAIN'] = 'test.gov.uk' unless ENV['GOVUK_APP_DOMAIN']
```

Comment:
@rubenarakelyan I've updated this to use `test.gov.uk` throughout. To get that to work I've had to make this change. I'm worried this might have unwanted side effects as this will be set for all `rake` tasks. Should probably be OK. What do you think? I'll squash this into the original commit before we merge if you're happy with it.